app/models/pair_request.rb

```diff
@@ -34,8 +34,9 @@ class PairRequest < ApplicationRecord
 
   validates :when,
     presence: true,
-    inclusion: { in: (Date.current..(Date.current + 1.month)),
-                 message: 'field must not be in the past or more than 1 month into the future' }
+    inclusion: { in: (Date.current..(Date.current + 0.months)),
```

Comment:
I assume the change to `0.months` from `1.month` was accidental? Seems to be the cause of the test failures in CI I think.

Reply:
Good catch, yes. Will fix!